src/modules/drawing/lib/style.js

```diff
@@ -18,6 +18,10 @@ const whiteSketchFill = new Fill({
     color: [255, 255, 255, 0.4],
 })
 
+const redFill = new Fill({
+    color: [255, 0, 0, 0.7],
+})
+
```

Comment:
Might make sense to start a new util file `src/utils/styleUtils.js` for generic styles